bigbluebutton-html5/imports/ui/components/app/styles.scss

```diff
@@ -152,10 +152,37 @@ $bars-padding: $lg-padding-x - .45rem; // -.45 so user-list and chat title is al
   flex: 1 100%;
   order: 2;
 
+  flex-direction: row;
+  position: relative;
+
   @include mq($small-only) {
     padding-bottom: $actionsbar-height;
     margin-bottom: $actionsbar-height;
-    position: relative;
+  }
+
+  @include mq($portrait) {
+    flex-direction: column;
+
+    @include mq($small-up) {
+      min-height: 60%;
```

Comment:
Closed-captions and the whole whiteboard container are inside this element. Why would it ever be 60%?